extension/src/js/background/index.js

```diff
@@ -219,22 +290,27 @@ function getTabAudio() {
             console.error("Error starting tab capture: " + (chrome.runtime.lastError.message || "UNKNOWN"));
             return;
         }
-        chrome.tabs.query({active: true, currentWindow: true}, function(tabs) {
-            var currTab = tabs[0];
-            if (currTab) { tabID = currTab.id; }
-
-        });
 
         let tracks = stream.getAudioTracks(); // MediaStreamTrack[], stream is MediaStream
-        let tabStream = new MediaStream(tracks);
-        window.audio = document.createElement("audio");
-        audioTagRef = window.audio; // save reference globally for volume control
-        window.audio.srcObject = tabStream;
-        window.audio.play();
-        localAudioStream = tabStream.clone();
```

Comment:
I used clone, so when the src muted the stream for listeners, the local audio playback would still be there. Right now, on muting the stream, it also mutes the local audio, which is not what we want